gammapy/modeling/models/temporal.py

```diff
@@ -166,7 +184,7 @@ def sample_time(self, n_events, t_min, t_max, t_delta="1 s", random_state=0):
 
         return t_min + time
 
-    def integral(self, t_min, t_max, oversampling_factor=100, **kwargs):
+    def integral(self, t_min, t_max, oversampling_factor=100, energy=None, **kwargs):
```

Comment:
In general the definition of only the energy center here is not sufficient. We would need `energy_min` and `energy_max` here as well and the perform the energy integration. However for the purpose of the feature is probably fine for now...

Reply:
That would again change the logic of the `MapEvaluator`, no? This implementation simply calculates the normalisation introduced at the specific (true) energy grid, which is then independently integrated over.